### PR TITLE
[DKP] Added supported Kubernetes versions to DKP apps table

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-89372

## Description of changes being made

Added a column on the supported Kubernetes versions. Also added other supported DKP catalog app versions.

### Preview

http://docs-d2iq-com-pr-4460.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/dkp-applications/index.md [hunk 1]
```diff
@@ -49,8 +49,11 @@ Follow these steps to install the DKP catalog from the CLI.
...
 ## Workspace DKP catalog applications
 
-| NAME                          | APP ID                |
-| ----------------------------- | --------------------- |
-| kafka-operator-0.20.0         | kafka-operator        |
-| spark-operator-1.1.6          | spark-operator        |
-| zookeeper-operator-0.2.13     | zookeeper-operator    |
+| Name                      | App ID             | Supported<br>Kubernetes<br>version |
+|---------------------------|--------------------|------------------------------------|
+| kafka-operator-0.20.0     | kafka-operator     | 1.21                               |
```
Comment (vespian) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/dkp-applications/index.md:54]: it's >=1.19

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/dkp-applications/index.md [hunk 2]
```diff
@@ -49,8 +49,11 @@ Follow these steps to install the DKP catalog from the CLI.
 
 ## Workspace DKP catalog applications
 
-| NAME                          | APP ID                |
-| ----------------------------- | --------------------- |
-| kafka-operator-0.20.0         | kafka-operator        |
-| spark-operator-1.1.6          | spark-operator        |
-| zookeeper-operator-0.2.13     | zookeeper-operator    |
+| Name                      | App ID             | Supported<br>Kubernetes<br>version |
```
Comment (gracedo) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/dkp-applications/index.md:52]: I think this is meant to be on one line

```suggestion
| Name                      | App ID             | Supported Kubernetes version |
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/dkp-applications/index.md [hunk 3]
```diff
@@ -49,8 +49,11 @@ Follow these steps to install the DKP catalog from the CLI.
...
+|---------------------------|--------------------|------------------------------------|
+| kafka-operator-0.20.0     | kafka-operator     | 1.21                               |
+| kafka operator-0.20.2     | kafka-operator     | 1.21<br>1.22                       |
```
Comment (gracedo) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/dkp-applications/index.md:55]: Why do we use a range in the 2.3 (`1.21 - 1.23`) table but use separate versions in 2.2 table (`1.21<br>1.22`)? Can we make these table formats consistent?

Comment (gvasquezvargas) [reply pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/dkp-applications/index.md]: I forgot there were patch releases between 1.21 and 1.22 hehehe. Thanks for pointing that out!

**Review by mmcelroy91 — APPROVED** [review]

**Review by alembiewski — APPROVED** [review]

**Pull request merged**